appserver-pipeline.yaml

```diff
@@ -0,0 +1,37 @@
+pr:
+  branches:
+    include:
+      - develop
+  paths:
+    include:
+      - AppserverTest/*
```

Comment:
Does this mean it only triggers when an AppserverTest changes?

Reply:
AFIAK it will trigger from the PR into develop, and include the AppserverTest contents every time it runs the script, regardless of if the actual unit tests have changed.